scripts/mkdoc.py

```diff
@@ -1,26 +1,24 @@
 #!/usr/bin/env python
+
 #
-# Script to generate basic documentation for CCPP metadata
+# Functions to generate basic documentation in HTML and LaTeX for CCPP metadata
 #
 
+# DH* TODO: create a Python module metadata.py with a class Metadata
+# and use this for ccpp_prebuild.py; create to_html and to_latex routines for it
+
```

Comment:
Please convert this into an issue.

Reply:
See above. For now we will proceed and start adopting good habits step by step.